sklearn/linear_model/tests/test_base.py

```diff
@@ -205,6 +206,25 @@ def test_linear_regression_sparse_multiple_outcome(random_state=0):
     assert_array_almost_equal(np.vstack((y_pred, y_pred)).T, Y_pred, decimal=3)
 
 
+def test_linear_regression_pd_sparse_dataframe_warning():
+    try:
+        import pandas as pd
+        if pd.__version__ < '0.24.0':
```

Comment:
You need to use LooseVersion or StrictVersion rather than string comparison.

Reply:
Done